crates/ironrdp-async/src/framed.rs

```diff
@@ -66,9 +66,13 @@ where
     S: StreamWrapper,
 {
     pub fn new(stream: S::InnerStream) -> Self {
+        Self::from_leftover(stream, BytesMut::new())
+    }
+
+    pub fn from_leftover(stream: S::InnerStream, leftover: BytesMut) -> Self {
```

Comment:
Maybe `with_leftower` would be a better name for a constructor? `from_` sounds like `leftover` should be the only argument, but we also have stream as a first argument. What do you think?

Reply:
You’re right. I’ll rename to `new_with_leftover` (`with_leftover` alone would suggest some sort of builder pattern)